src/sentence-splitter.ts

```diff
@@ -194,7 +194,8 @@ export function splitAST(paragraphNode: TxtParentNode): TxtParentNode {
     }
 
     // It follow some text that is not ended with period.
-    splitParser.close(anyValue);
+    // TODO: space is correct?
+    splitParser.close(space);
```

Comment:
close handle following
```
<p>text     </p>
             ~~~
```

This is a parts of a Sentence OR isolate WhiteSpace?